.github/workflows/verify.yaml

```
@@ -24,9 +24,9 @@ jobs:
           check-latest: true
 
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@051d91933864810ecd5e2ea2cfd98f6a5bca5347 # v6.3.2
+        uses: golangci/golangci-lint-action@0adbc47a5910e47adb692df88187ec8c73c76778 # v6.4.0
         with:
-          version: v1.63
+          version: v1.64.5
```

keep 
```suggestion
          version: v1.64
```

then it will pick all patch and we dont need to update